services/ui-src/src/utils/validation/schemas.ts

```diff
@@ -124,7 +124,7 @@ export const radioOptional = () => radio().notRequired();
 
 // DYNAMIC
 export const dynamic = () =>
-  array().min(1).of(text()).required(error.REQUIRED_GENERIC);
+  array().min(1).of(mixed()).required(error.REQUIRED_GENERIC);
```

Comment:
Blocking: We should actually update this now so the validation works. Something like this should do the trick:
```suggestion
  array().min(1).of(object().shape({
    id: text(),
    name: text(),
  })).required(error.REQUIRED_GENERIC);
```